android/app/src/main/res/layout/forecast_timestep.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/forecastTimeStep"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="6dp"
+    android:layout_marginBottom="6dp"
+    android:orientation="horizontal">
+
+    <LinearLayout android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+        <TextView
+            android:id="@+id/timeTextView"
+            style="@style/time"
+            android:layout_width="wrap_content"
```

Comment:
Wrong color in dark theme:
![Screenshot 2024-12-12 at 8 36 51](https://github.com/user-attachments/assets/e7d64101-bbcd-45cf-b60c-5fdf60e53a28)